### ISSUE TITLE
The text color of formattable htmlwidget is too light in R Notebook

### ISSUE THREAD

**Issue by renkun-ken** [renkun-ken/formattable#68]
The latest [RStudio Preview](https://www.rstudio.com/products/rstudio/download/preview/) has supported R Notebook (enable by Global Options -> R Markdown -> Enable R notebook).

However, the color of text in a formattable htmlwidget is too light to see. For example,

```r
library(formattable)
formattable(mtcars, list(mpg = color_tile("white", "lightpink")))
```

![formattable-htmlwidget-r-notebook](https://cloud.githubusercontent.com/assets/4662568/15796740/a56e2d12-2a36-11e6-8961-bfc301bf46ba.png)

@timelyportfolio, any idea on how to fix this?

**Comment by timelyportfolio** [comment]
Interesting.  Hopefully, I will check it out in the morning.

**Comment by renkun-ken** [comment]
It works fine with light RStudio theme like TextMate:

![formattable-htmlwidget-r-notebook2](https://cloud.githubusercontent.com/assets/4662568/15797271/4eee2d94-2a44-11e6-88ef-4be390aea8b8.png)

**Comment by renkun-ken** [comment]
Posted an issue at RStudio: https://support.rstudio.com/hc/en-us/community/posts/211637527-The-text-color-of-htmlwidget-can-be-too-light-in-R-notebook-under-dark-themes

**Comment by timelyportfolio** [comment]
It appears RStudio is assigning a `body` color which is impacting the table.

![image](https://cloud.githubusercontent.com/assets/837910/15802919/97dc7f8e-2a89-11e6-8ef9-abf5b593b88d.png)

I'm not sure internally how RStudio assigns this color.  We could add some code, but I think it would better handled on the RStudio end.  I wonder what other `htmlwidgets` might be affected.  I'll keep on a dark theme to see what other issues show up.

![image](https://cloud.githubusercontent.com/assets/837910/15802923/fa01dfba-2a89-11e6-9c3f-d5992c744a57.png)

**Comment by kevinushey** [comment]
Indeed, RStudio hard-codes the background color as `white` at the moment,  but we should update / set this dynamically based on the theme / HTML widget specification. I'll log this on our internal tracker and try to get this cleaned up before the RStudio release.

**Comment by iozeroff** [comment]
I'm still having this problem! Dark mode is formatting my table font color to white.

**Comment by kmshort** [comment]
> I'm still having this problem! Dark mode is formatting my table font color to white.

Same here -- funny, after no posts in 4 years, two of us post in 18 minutes!

**Comment by ninadicara** [comment]
Ran into this issue too in a markdown doc and ended up here -  I'm displaying it in the Viewer pane as a work around so I can see the table output without changing my theme (and without pushing all output to Viewer)😄 

In case helpful for anyone else:
```
library(formattable)
library(htmltools)

mytable <- formattable(mtcars, list(mpg = color_tile("white", "lightpink")))
html_print(as.htmlwidget(mytable))
```

**Comment by mexiro** [comment]
The issue is open on rstudio github.
https://github.com/rstudio/rstudio/issues/6489
